html/textroomtest.js

```diff
@@ -153,16 +155,12 @@ $(document).ready(function() {
 										$('#chatroom').get(0).scrollTop = $('#chatroom').get(0).scrollHeight;
 									} else if(what === "leave") {
 										// Somebody left
-										var username = json["username"];
-										var when = new Date();
```

Comment:
Why has `username` been removed? This is a remote user that's gone, and we need that ID to remove the element as the following lines show.

Reply:
Yes my fault, there was a `var username` a couple of lines before.
Fixing it.